src/BackpackManager.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Backpack\CRUD;
+
+use Backpack\CRUD\app\Http\Controllers\Contracts\CrudControllerContract;
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanel;
+use Illuminate\Support\Facades\Facade;
+
+final class BackpackManager
```

Comment:
What does this manage in Backpack... only CRUD Panels, right? So then... perhaps we should call it `CrudPanelManager` instead?

Think about it this way... if we have 3 separate packages: `backpack/ui`, `backpack/auth` and `backpack/crud`... where would this live? Inside the CRUD package right? So then it's an over-reach to say it's a BackpackManager, it's actually a `CrudPanelManager`.